docker-compose.yml

```diff
@@ -30,6 +30,8 @@ services:
     image: mongo
     restart: always
     user: "${UID}:${GID}"
+    ports:
```

Comment:
These ports cannot be exposed by default. if this is needed, it has to be done with the override compose file: https://docs.librechat.ai/install/configuration/docker_override.html